packages/commons-ui-core/package.json

```
@@ -0,0 +1,62 @@
+{
+  "name": "commons-ui-core",
+  "version": "2.0.0-alpha.0",
+  "author": "CodeForAfrica",
+  "license": "GPL-2.0-or-later",
+  "private": true,
```

```suggestion
  "private": false,
```
We need to publish this